examples/sample-pub/metering.js

```diff
@@ -73,7 +76,7 @@ const MeteringDemo = {
    * so it lives past 7 days in Safari.
    * https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/
    */
-  registerUser: (gaaUser) => {
+  registerUser: async (gaaUser) => {
```

Comment:
Doesn't look like this needs to be async.

Reply:
Removed 👍